app/controllers/api/v1/tasks_controller.rb

```diff
@@ -0,0 +1,37 @@
+class Api::V1::TasksController < ApplicationController
+  before_action :load_and_authorize_resource, except: :create
+
+  def create
+    @task = Task.new(task_params)
+    authorize @task
+    if @task.save
+      render json: TaskSerializer.new(@task).serialized_json, status: :created
+    else
+      render json: @task.errors, status: :unprocessable_entity
+    end
+  end
+
+  def update
+    Tasks::UpdatePositionService.new(@task, task_position_param).call if task_position_param
```

Comment:
it's not correct, this service should choose type of update - by position or another params